### ISSUE TITLE
Create SVG on Adobe Illustrator

### ISSUE THREAD

**Issue by chiaralogan** [maxwellito/vivus-instant#17]
Hi!
I've a problem that I can't solve.
I'm trying to export SVG for Vivus from Adobe Illustrator, I've tried all possible options but Visus does not animate the SVG.
I can't understand where the error is, can you help me please?
Thanks, have a nice day! :)

**Comment by maxwellito** [comment]
Hi there,

Can you please share your SVG? :)
Also, to remind you, vivus animate stroke based SVG. So if your SVG is only
made of filled shapes, no animation will happen.

**Comment by chiaralogan** [comment]
Hi, this is one of the millions of tests that I do ;)
[prova 5.svg.zip](https://github.com/maxwellito/vivus-instant/files/4890108/prova.5.svg.zip)

**Comment by maxwellito** [comment]
Yes, that's what I was afraid of. Your SVG doesn't use strokes, but filled shapes. Vivus animate lines (or shape borders if you prefer).

_Here is one of the shape of your SVG_
_You can see your shapes are filled, but no stroke._

![Screen Shot 2020-07-08 at 11 40 47](https://user-images.githubusercontent.com/1097232/86917661-fe68a900-c114-11ea-820e-f9246359d630.png)

_Here is of the edited SVG_
_In this case the shapes are using strokes, no filling_

![Screen Shot 2020-07-08 at 11 42 13](https://user-images.githubusercontent.com/1097232/86917678-07f21100-c115-11ea-9ef7-6d2bb59838fc.png)

Here is the edited SVG if you want to play with it and see the difference. 
[prova 5.svg.zip](https://github.com/maxwellito/vivus-instant/files/4890563/prova.5.svg.zip)

You can also play with the [SVG used for the manual tests of Vivus](https://github.com/maxwellito/vivus/tree/master/test/manual)

I hope this helps :)
